components/BreedCheckbox/BreedCheckbox.tsx

```diff
@@ -16,6 +16,15 @@ export const BreedCheckbox = ({ breed, handleCheckbox, isChecked }: Props) => {
     handleCheckbox(event.target.value);
   };
 
+  const formatBreed = () => {
+    if (!breed.includes("/")) return breed;
+
+    return `${breed.slice(0, breed.indexOf("/"))}-${breed.slice(
+      breed.indexOf("/") + 1,
+      breed.length
+    )}`;
```

Comment:
Not much readable. What about something like this:

```suggestion
	const [subBreed, breed] = breed.split('/');
    return `${subBreed} ${breed}`;
```